src/it/resources/com/google/checkstyle/test/chapter4formatting/rule6210chainindentation/ClassWithChainedMethods.java

```diff
@@ -0,0 +1,36 @@
+package com.google.checkstyle.test.chapter4formatting.rule6210chainindentation;  //indent:0 exp:0
+
+public class ClassWithChainedMethods { //indent:0 exp:0
```

Comment:
> 6210

Please quote text from 4.6.2 - https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.6.2-horizontal-whitespace that is matching chaining.

I think you need to move this file to rest Indentation test inputs 
https://github.com/checkstyle/checkstyle/tree/master/src/it/java/com/google/checkstyle/test/chapter4formatting/rule4841indentation

Please name this file as "ClassWithChainedMethodsCorrect" and create one more file "ClassWithChainedMethods" that will have violations ("//warn" comments). We need positive and negative tests.

Reply:
done